torch/csrc/autograd/variable.cpp

```diff
@@ -384,6 +384,13 @@ unsigned VariableHooks::_register_hook(const Tensor& self, std::function<Tensor(
 }
 
 void handle_view_on_rebase(DifferentiableViewMeta* diff_view_meta, bool indirect) {
+  // TODO: Remove this warning once we allow XLA to workaround CopySlices.
+  if (diff_view_meta->base_.device().type() == c10::DeviceType::XLA) {
+    TORCH_WARN("Backward through inplace update on view tensors is WIP for XLA backwend. "
```

Comment:
I would formulate this slightly differently maybe and adapt to the `indirect` flag:
- When indirect=False, this is called because you are modifying inplace (right now) a view that requires gradients.
- When indirect=True, this is called because another view of the same base was modified inplace (in the past) and this view requires gradients.